src/app/docs/toolbar/toolbar-docs.component.html

```diff
@@ -0,0 +1,147 @@
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/toolbar/list-toolbar-context.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Record Toolbar"
+      >
+        <p>
+          The record toolbar appears below the <a href="https://developer.blackbaud.com/skyux/components/page-summary">page summary</a> on record pages. It contains actions to manipulate the record. Place frequently used actions directly in the toolbar, and place less-frequently used actions in a more actions menu. Actions to collapse and expand all tiles on the record are docked on the right side of the toolbar.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/toolbar/page-toolbar-context.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Tile toolbar"
+      >
+        <p>
+          The tile toolbar appears in a <a href="https://developer.blackbaud.com/skyux/components/tiles">tile</a> above the control it affects. It contains actions to manipulate the contents of the tile or a portion of the tile. Only place the most important actions directly in the toolbar because tiles have limited horizontal real estate. Place all other actions in a more actions menu. If a tile requires no more than two filters, the filters should be docked on the right side of the toolbar.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/toolbar/tile-toolbar-context.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Composition"
+    >
+      <sky-docs-design-guideline-thumbnail
+        imageSource="~/assets/img/guidelines/toolbar/anatomy.png"
+      >
+      </sky-docs-design-guideline-thumbnail>
+
+      <sky-docs-design-guideline
+        heading="Standard toolbar actions"
+      >
+        <p>
+          Toolbars won't always need every possible button, but when present, buttons should be in the expected locations. Standard actions should appear from left to right in the following order when present in a toolbar:
+        </p>
+
+        <sky-toolbar-guidelines></sky-toolbar-guidelines>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Responsiveness"
+    >
+      <sky-docs-design-guideline
+        heading="Toolbar action wrapping"
+      >
+        <p>
+          The actions in a toolbar (including the more actions button) shouldn't extend beyond 50% of the screen. If the actions would extend beyond 50% of the screen, the rightmost action should collapse into the more actions menu.
```

Comment:
I'd change the first sentence to: `Do not extend the standard actions and more actions menu beyond 50 percent of the container.` (Most of this is just tightening up, so no worries if you disagree, but I think `screen` in the current version is the wrong word.)

I'd change the second sentence to: `Instead, collapse the rightmost actions into the more actions menu.`

Reply:
Following up again. Do you think these changes are worth making, or should we just resolve the conversation?